packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx

```diff
@@ -33,6 +33,9 @@ const SearchField = forwardRef(({ placeholder, mapsindoors, results, clicked, cl
         },
         focusInput() {
             elementRef.current.focusInput();
+        },
+        clearInput() {
```

Comment:
We could rename it to reflect the "original" method name on the <mi-search>, "clear".

Reply:
Done ✅